### PR TITLE
Created contact form section: The contact form section has been created in index.html file and the style code is added in style.css file. The changes are pushed in created_contact_form_section branch to get merge with main

### PR DESCRIPTION
**Pull request by mirwaisfarahi**
The contact form has been created with the following specifications:
1. Name field accepts up to 30 chars.
2. Email field accepts only emails.
3. Message field accepts up to 500 chars.
4. All fields have placeholder text.
5. All fields have the correct type.
6. A user cannot submit the form if one or more of the fields is empty.

The contact form with full name, email, message and submit button fields are pushed to the created_contact_form section. This pull request will be used to merge it with the main branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -188,8 +188,38 @@ <h3 id="h3">LET'S CONNECT</h3>
                     </ul>
                 </div>
             </section>
-
         </main>
+
+        <footer>
+            <section class="contact-form">
+                <h1 id="contact-me">Contact me </h1>
```
Comment (OmarRamoun) [inline index.html]: - [x] [REQUIRED] Please, make sure not to use more than 1️⃣ one `h1` tag in the whole document for semantic puposes.

File: index.html [hunk 2]
```diff
@@ -188,8 +188,38 @@ <h3 id="h3">LET'S CONNECT</h3>
...
+                <p id="contact-text">
+                    If you ave an application you are interested in developing, a feature that you
+                    need built or a project that needs coding. I'd love to help with it.
+                </p>
+                <form>
```
Comment (OmarRamoun) [inline index.html]: - [x] [REQUIRED] Kindly, make the Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html) 🔗.

File: style.css [hunk 3]
```diff
@@ -329,3 +329,111 @@ p {
   border-top: 1px solid #dfe1e6;
   margin-top: 20px;
 }
+
+footer {
+  display: flex;
```
Comment (OmarRamoun) [inline style.css]: - [x] [OPTIONAL] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

image

### In our case

you can easily use a `.flex` class in multiple places .

- (your code)

```css
.footer {
  /* ...code */
  display: flex;
  /* ...code */
}

.contact-form {
  /* ...code */
  display: flex;
  /* ...code */
}
```

- suggestion

```css
.flex {
  display: flex;
}
```

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @mirwaisfarahi 👋

Good job so far! 👏. Your Coding Style is Impressive! 💪.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** 🌟 that can improve your code quality 💯 and help you **get a better job** 💼.

_I will guide 📔 you through fixing 🔧 these bugs 🐛, just **follow my comments down below** 🔽 and make sure you **don't miss any** of them, in order to get your project approved ✅_

## Highlights 🌟


- ✔️ descriptive commit messages
- ✔️ Very descriptive PR title and description
- ✔️ used GitHub flow
- ✔️ ✔️ the UI Design is very close to the Figma Design (Good Job 👍 👏 )
- The code is top-notch! 💯 and have a huge potential.

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. So, you can skip them if you don't have the time but remember to come back and implement them because they are going to enhance the quality of your code_

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -136,7 +136,7 @@ <h3>Multi-Post Stories</h3>
                 </div>
             </section>
 
-            <section class="about">
+            <section class="flex col-direction about">
                 <h1>About<br> Myself</h1>
```
Comment (sja-thedude) [inline index.html]: - [x] As suggested by the previous reviewer. Kindly make sure to use only one h1 tag in the entire document. You have already used h1 tag on line 29. Make sure to remove this h1 tag from "About Myself" and replace with h2 or h3.

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html]: Good job in implementing the changes 👍

File: style.css (hunk 3 above)
Comment (sja-thedude) [reply style.css]: Good job in implementing the changes 👍

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @mirwaisfarahi, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job in implementing the changes so far with adding formspree to the contact form.

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @mirwaisfarahi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**